solr/modules/otel/README.md

```diff
@@ -0,0 +1,33 @@
+Apache Solr Open Telemetry Tracer
+=====================================
+
+Introduction
+------------
+This module brings support for the new [OTEL](https://opentelemetry.io) standard,
+and exposes a tracer configurator that can be enabled in the
+`<tracerConfig>` tag of `solr.xml`:
+
+```xml
+<tracerConfig name="tracerConfig" class="org.apache.solr.otel.OtelTracerConfigurator" />
```

Comment:
Yes, I agree to change module and package name. But I like a descriptive class name too, since `TracerConfigurator` is the abstract class name. So if package name is `opentelemetry` i think class name can safely be `OtelTracerConfigurator`.